src/app/App.svelte

```diff
@@ -31,33 +31,36 @@
 	<div class="logo">
 		<!-- eslint-disable-next-line svelte/no-at-html-tags -->
 		{@html logo}
-		<span>
-			{#if import.meta.env.GIT_TAG && import.meta.env.GIT_COMMIT}
-				<a
-					href="https://github.com/quoid/userscripts/releases/tag/{import.meta
-						.env.GIT_TAG}"
-				>
-					{import.meta.env.GIT_TAG}
-				</a>
-				(<a
-					href="https://github.com/quoid/userscripts/commit/{import.meta.env
-						.GIT_COMMIT}"
-				>
-					{import.meta.env.GIT_COMMIT.slice(0, 7)}
-				</a>)
-			{:else}
-				<span>{version}</span>
-				<span>{build}</span>
-			{/if}
-		</span>
 	</div>
+	<div class="version">
+		{#if import.meta.env.GIT_TAG && import.meta.env.GIT_COMMIT}
+			<a
+				href="https://github.com/quoid/userscripts/releases/tag/{import.meta.env
+					.GIT_TAG}"
+			>
+				{import.meta.env.GIT_TAG}
+			</a>
+			(<a
+				href="https://github.com/quoid/userscripts/commit/{import.meta.env
+					.GIT_COMMIT}"
+			>
+				{import.meta.env.GIT_COMMIT.slice(0, 7)}
+			</a>)
+		{:else}
+			<span>{version}</span>
+			<span>{build}</span>
+		{/if}
+	</div>
+	<br />
 	<p>
-		You can turn on the Userscripts iOS Safari extension in Settings.
+		You can turn on the Userscripts iOS Safari extension in Settings or Safari,
+		then use the extension in Safari.
+		<br />
 		<a
 			href="https://github.com/quoid/userscripts/blob/{import.meta.env
 				.GIT_TAG ?? 'main'}/README.md"
 		>
-			Read the docs.
+			Read online documentation
```

Comment:
```suggestion
			Read the documentation online.
```

If you really want to change this, I think this phrasing reads better in English.